lnwire/channel_update.go

```diff
@@ -54,7 +54,12 @@ type ChannelUpdate struct {
 	// corresponds to the first node in the previously sent channel
 	// announcement and 1 otherwise. If the second bit is set, then the
 	// channel is set to be disabled.
-	Flags ChanUpdateFlag
+	ChannelFlags ChanUpdateFlag
+
+	// MsgFlags is a bitfield that describes additional meta-data concerning
+	// how the update is to be interpreted. Currently, the least-significant
+	// bit must be set to 1 if the optional field MaxHtlc is present.
+	MsgFlags ChanUpdateFlag
```

Comment:
suggestion: name this `MessageFlags` to reflect what's in the spec?

Reply:
fixed